tests/configuration/verifyPMMSettingsPageFunctionality_test.js

```diff
@@ -348,6 +348,21 @@ Scenario(
     scheduledPage.openScheduledBackupsPage();
   },
 );
+
+Scenario(
+  'PMM-11640-backup-managment-enabled-by-default @backup',
```

Comment:
Done

Reply:
Not done. It still doesn't cover the "Backups" menu icon check. Or https://jira.percona.com/secure/Tests.jspa#/testCase/PMM-T1659.

Reply:
Icon check blocked by this defect - https://jira.percona.com/browse/PMM-11832 - better to wait for a fix since we don't have this icon in tests/pages/leftNavMenu.js POM at all. 
Regarding the upgrade scenario - that is not under my responsibility and should be handled in a different PR by the already assigned eng here: https://perconacorp.slack.com/files/T03FRR0BJ/F04UREZDUR1?origin_team=T03FRR0BJ

Reply:
Check for BM Icon on the homepage was added